### ISSUE TITLE
htmlscraper needs additional headers

### ISSUE THREAD

**Issue by sogehige** [ckatzorke/howlongtobeat#17]
There needs to be added additional headers into htmlscraper.ts. Without it I got 403 response.

```javascript
  headers: {
    Referer: 'https://howlongtobeat.com/',
    'Content-type': 'application/x-www-form-urlencoded',
    'User-Agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/88.0.4324.150 Safari/537.36',
  },
```

I would do a PR but not sure how dist and src is done in your project

**Comment by ChronicLynx** [comment]
Agree, there does appear to be a problem currently.

**Comment by ckatzorke** [comment]
Hey, thank you.
Yes, the referrer is not necessary, and true, it would be awkward to set it to howlongtobeat.com

I added a random user-agents, the package is available as v1.3.1.
All integration-tests have successfully passed, so it should work now
